<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EdgeSettingsWidget</class>
 <widget class="QWidget" name="EdgeSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
   <item>
    <widget class="QToolButton" name="rotationButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="clockwizePage">
      <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0,1" columnminimumwidth="0,35,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="2">
        <widget class="QSlider" name="clockStringSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="clockStringValLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSlider" name="clockFretSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="clockFretValLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="fretLabel">
         <property name="text">
          <string>Fret</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="stringLabel">
         <property name="text">
          <string>String </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="antiClockwizePage">
      <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,0" columnstretch="0,0,0,1" columnminimumwidth="0,35,0,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="1" colspan="2">
        <widget class="QLabel" name="antiStringValLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLabel" name="antiFretValLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QSlider" name="antiFretSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSlider" name="antiStringSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="fretLabel_2">
         <property name="text">
          <string>Fret</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="stringLabel_2">
         <property name="text">
          <string>String </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
